sycl/doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc

```diff
@@ -197,6 +197,11 @@ class device_global {
 
 _T_ is restricted to types that have a trivial destructor. _PropertyListT_ enables properties to be associated with a `device_global`.
 
+[NOTE]
+====
+If _T_ is prepended `const` then in the cuda and hip backends the compiler will use the constant address space for the `device_global` instead of the global address space. This allows for optimizations such as using the Nvidia constant cache, and is equivalent to usage of `$$__constant__$$` in the CUDA and HIP native programming languages.
+====
```

Comment:
I think this should be reworded slightly to align with some of the other non-normative notes: it's useful to separate what the specification allows from specific behavior of DPC++.

So, maybe we could say something like:

```suggestion
[NOTE]
====
If _T_ is `const` then implementations may choose to allocate the `device_global` in a dedicated constant address space as an optimization. When using the {dpcpp} compiler with the CUDA or HIP backend, declaring a `device_global<const T>` is equivalent to declaring a `$$__constant__$$` variable.
====
```